### ISSUE TITLE
sessionName issue

### ISSUE THREAD

**Issue by carltesta** [NikolaySuslov/aframe-croquet-component#1]
This is an amazing component! I'm wondering if the sessionName is arbitrary or whether I have to use a specific one. When I change the sessionName to something besides "demo-scene" it either works momentarily and then breaks or doesn't work at all. Do sessionNames have to be registered somewhere other than the Aframe HTML?

**Comment by NikolaySuslov** [comment]
Thanks for the interest and pointing to this! 
Yes, the latest Croquet SDK now supports auto generated session names in url paths like `/?q=randomSessionName`. Thus, there are no need to hardcode session names in html anymore. The users should share generated links between each other to connect. So, I have updated source code to use that latest feature.
Hope that will help.

Best regards,
Nikolai

**Comment by carltesta** [comment]
Ah, fantastic. Thank you for the quick response. I see now and it works as expected. Is there a way to hard code the session name? For example, if I wanted every user who visited index.html to be in the same session without specific users needing to share their own url?

**Comment by NikolaySuslov** [comment]
Yes, sure. I will look at this more closer, as user defined session names may require session/snapshot cleanup tracking on application init.

**Comment by NikolaySuslov** [comment]
In the latest update, I fixed syncing issues and revert option for manual setting of a session name just in the component.  Password should be also specified along the name.
```html
<a-scene croquet="sessionName: test; password: test">
```
